addon/services/-observer-admin.js

```diff
@@ -220,4 +220,21 @@ export default class ObserverAdmin extends Service {
     }
     return true;
   }
+
+  /**
+   * Stringify observerOptions for use as a key.
+   * Excludes observerOptions.root so that the resulting key is stable
+   *
+   * @param {Object} observerOptions
+   * @private
+   * @return {String}
+   */
+  _stringifyObserverOptions(observerOptions) {
+    let replacer = (key, value) => {
+      if (key === 'root') return undefined;
```

Comment:
We could stringify the `String` version of `scrollableArea` by passing it somehow from the mixin.

The scenario I am thinking of is 2 scrollable areas - An `<aside>` that has a defined scrollable area and a main that just defaults to the document viewport or a different scrollable area.  However, both have the same rootMargin && threshold.  Not a big deal though so if you disagree or adds too much complexity, I'm fine with this!

https://github.com/DockYard/ember-in-viewport/blob/master/addon/mixins/in-viewport.js#L158